src/System/Interactions.jl

```diff
@@ -372,6 +373,107 @@ function set_forces_aux!(B, dipoles::Array{Vec3, 4}, ints::Interactions, i::Int,
     end
 end
 
+function set_forces_aux_homog!(B, dipoles::Array{Vec3, 4}, buf::Array{Vec3,4}, ints::Interactions, i::Int, sys::System{N}) where N
```

Comment:
Big picture question -- is there a way to get the performance benefits while avoiding the code duplication? Perhaps we can discuss this on a Zoom call.

Reply:
I think so! This is (at least related to) the "possible low hanging fruit." We can discuss sometime this week after the Monday holiday